.dependabot/config.yaml

```diff
@@ -14,6 +14,12 @@ update_configs:
     ignored_updates:
       - match:
           dependency_name: none
+  - package_manager: python
```

Comment:
Don't you removed duplicated config for python in other projects today ?
Or I miss something.

Reply:
+1 Your right :-)